api/src/gameState/gameState.controller.ts

```diff
@@ -12,12 +12,13 @@ import {
   HttpStatus,
   BadRequestException,
   ConflictException,
+  UseGuards,
 } from '@nestjs/common';
 import { AppGateway } from '../app.gateway';
 import { GameState, Player, Node } from '../types';
 import { GameStateService } from './gameState.service';
-import { INITIAL_BOARD } from '../constants';
 import { Game } from '@prisma/client';
+import { JwtAuthGuard } from '../auth/jwt-auth.guard';
 
 @Controller('games')
```

Comment:
Can't we add the `UseGuards` on the controller instead of adding it on each route? As we have to securise all the routes of the controller

Reply:
cleaver !